mvnw

```diff
@@ -50,7 +50,7 @@ fi
 cygwin=false;
 darwin=false;
 mingw=false
-case "`uname`" in
+case "$(uname)" in
```

Comment:
I was going to suggest to push this change upstream, but I believe that it's already applied there: https://github.com/apache/maven-wrapper/blame/maven-wrapper-3.3.2/maven-wrapper-distribution/src/resources/mvnw

Should we update our bundled Maven Wrapper (rather than manually modifying it)?

Reply:
Probably, though that feels like a distinct PR.

Reply:
Ack: https://github.com/igniterealtime/Openfire/pull/2593